R/expand_design_redesign.R

```diff
@@ -36,7 +37,7 @@
 #' }
 #'
 #' @export
-expand_design <- function(template, ..., expand = TRUE, prefix = "design") {
+expand_design <- function(designer, expand = TRUE, prefix = "design", ...) {
```

Comment:
you moved the dots back?

Reply:
thx - damnit, my fault, this was merging in changes from master. will fix.